src/pages/tBTC/index.tsx

```diff
@@ -1,12 +1,15 @@
-import PageLayout from "../PageLayout"
 import { PageComponent } from "../../types"
 import HowItWorksPage from "./HowItWorks"
 import TBTCBridge from "./Bridge"
 import { featureFlags } from "../../constants"
 import { ExplorerPage } from "./Explorer"
+import { Outlet } from "react-router-dom"
+import useDocumentTitle from "../../hooks/useDocumentTitle"
 
 const MainTBTCPage: PageComponent = (props) => {
-  return <PageLayout title={props.title} pages={props.pages} />
+  const { title = "tBTC" } = props
```

Comment:
Let's destructure the `title` prop in the component parameters:

```
const MainTBTCPage: PageComponent = ({ title = "tBTC" }) => {
(...)
```

Reply:
Ref commit: ea6140e3